sdk/communication/communication-verified-recipients/README.md

```diff
@@ -0,0 +1,192 @@
+Verification README
+
+# Azure Communication Verified Recipients client library for JavaScript
+
+The Verified Recipient library allows users to verify numbers for send messages or calling.
```

Comment:
Maybe?
```suggestion
The Verified Recipient library allows users to verify numbers for sending messages or calling.
```

Reply:
Could I put, "The Verified Recipient library requires users to verify numbers before sending messages or calling them."?
or is that still unclear?

Reply:
sounds good